bin/test/main.ml

```diff
@@ -291,8 +298,16 @@ let run_exn (`Setup ()) (`Non_deterministic non_deterministic)
     match active, non_deterministic, Block.mode t, Block.value t with
     (* Print errors *)
     | _, _, _, Error _ -> Block.pp ?syntax ppf t
-    (* Skip raw blocks. *)
-    | true, _, _, Raw -> Block.pp ?syntax ppf t
+    (* Skip or copy raw blocks. Without parts support *)
+    | true, _, _, Raw -> 
+    (match Block.part t with
```

Comment:
I would probably swap the two nested pattern matching here as the error mentions files (as in the ones passed to the `file` label) and you can still get that error if you just wrote:
```
\```scheme part=2
\```
```
We can also make the error a bit more generic, "Parts are not supported for non-OCaml code blocks".
What do you think?

Note that I'm just nit-picking here as I believe the error should actually be reported at parsing time as we shouldn't allow some labels on some blocks anyway but that's a bit out of scope for this PR.

Reply:
Option 2: 
> We can also make the error a bit more generic, "Parts are not supported for non-OCaml code blocks".

Implemented in 5ae04dd